src/couch_replicator/src/couch_replicator_api_wrap.erl

```diff
@@ -898,39 +899,7 @@ json_to_doc_info({Props}) ->
             }
     end.
 
-bulk_results_to_errors(Docs, {ok, Results}, interactive_edit) ->
```

Comment:
Seems like this was dead code left over from a previous change?

Reply:
Yeah, it was code from when we had local end points and made couch_db:update_docs(...) calls directly